catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -370,49 +372,77 @@
         </reference-listener>
     </reference-list>
 
-    <!-- create the ddf bean -->
-    <bean id="ddf" class="ddf.catalog.impl.CatalogFrameworkImpl">
+    <!--
+        Framework properties bean
+    -->
+    <bean id="frameworkProperties" class="ddf.catalog.impl.FrameworkProperties">
```

Comment:
I wonder if we can even break this blueprint file into multiple blueprint files.